tests/test_assets.py

```diff
@@ -38,9 +43,14 @@ def test_get_asset(client: Client):
     assert 'content' in response, 'Missing content in response'
 
 
-def test_update_asset(client: Client):
+@pytest.mark.parametrize('name,description', [
+    ('foo', ''),
+    ('', 'foo'),
+    ('foo', 'bar'),
```

Comment:
is it an idea to test all cases? explicit None, str and not provided?

Reply:
None fails for unknown reasons... but could add not provided